dev/CommonStyles/AppBarButton_rs5_themeresources.xaml

```diff
@@ -64,8 +123,12 @@
                         Background="{TemplateBinding Background}"
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
-                        CornerRadius="{TemplateBinding CornerRadius}" >
-
+                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}" >
+                        <contract5NotPresent:Grid.Resources>
```

Comment:
A comment here would be helpful -- in RS3 what bug did we fix that makes this unnecessary?

Reply:
Add comment before the style. Otherwise script may have problem to handle it.